spec/lib/embed/viewer/file_spec.rb

```diff
@@ -51,6 +51,13 @@
       html = Capybara.string(file_viewer.to_html)
       # visible false because we display:none the container until we've loaded the CSS.
       expect(html).to have_css '.sul-embed-file-list', visible: false
+      expect(html).to have_css 'a[download]', visible: false, text: '12.35 kB'
+    end
+    it 'with no file size' do
```

Comment:
I could, but part of me feels that rspec does this for us. For example with this test in a failure it reads:

```sh
Failures:

  1) Embed::Viewer::File body_html with no file size
     Failure/Error: expect(html).to have_css 'a[download]', visible: false, text: /^Download$/
       expected #has_css?("a[download]", {:visible=>false, :text=>/^Download$/}) to return true, got false
     # ./spec/lib/embed/viewer/file_spec.rb:61:in `block (3 levels) in <top (required)>'
```

And the hope is that the Rspec formatting should provide this to a developer?

What do you think?